package.json

```diff
@@ -147,6 +147,7 @@
     "lodash": "^4.17.21",
     "luxon": "^3.2.1",
     "map-cursor-to-max": "^1.0.0",
+    "mathjs": "^14.0.0",
```

Comment:
this will be removed once we set everything on the metaphysics, and potentially later on backend